### ISSUE TITLE
[question] How to get the texture projection right?

### ISSUE THREAD

**Issue by Rabbit-sk** [alicevision/Meshroom#1722]
**Problem description**
The texture is projected not only on the first visible part of the model surface, but also on all model surfaces that are not visible from the point of view of the given camera.

**Screenshots**
Draft mesh
![obrázok](https://user-images.githubusercontent.com/74265320/177845976-d1a5f79a-dc76-4840-b06c-16e19941d0c0.png)
![obrázok](https://user-images.githubusercontent.com/74265320/177847517-d044c2c6-dcdc-483d-aede-de9a3a448ebe.png)

**Dataset**
I know, there is strong direct light, but it is for testing purposes.
![obrázok](https://user-images.githubusercontent.com/74265320/177848487-103ca79e-050b-4d87-bef1-bdd0cb0b3082.png)

**Texture**
![obrázok](https://user-images.githubusercontent.com/74265320/177853943-754e4b67-22cd-4526-baf4-83323dda9ecc.png)

**SW and OS info**
Version 2021.1.0
Windows 10 
Python 3.7.4

**Additional context**
I used the ConvertSFMformat node to pass the filtered SFM data to the Texturing node to achieve texturing with only one image.

**Comment by fabiencastan** [comment]
Thanks for the feedback. Could you share the inputs of your Texturing node to replicate the problem?

**Comment by Rabbit-sk** [comment]
Of course, but I prefer sending the mesh by email. Could you please give me your email address?

**Comment by Ben-Mack** [comment]
I'm having exactly the same problem of texture projected through the first visible part of model (please see the OP screenshot), what is the conclusion of this issue? Are there some config or any direction/idea to solve this? @fabiencastan  @Rabbit-sk 
(I guess it's related to Visibility Remapping Method in Texturing node but I didn't have expected result after tested many different config in Texturing node)

**Comment by fabiencastan** [comment]
Hello, sorry this is an old issue, I don't remember exactly and I did not manage to retrieve the corresponding data to test it again.
Could you share a dataset?
My email: fabien.castan@technicolor.com
